python/tests/unit/arcticdb/version_store/test_normalization.py

```diff
@@ -437,3 +437,10 @@ def test_columns_names_series_dynamic(lmdb_version_store_dynamic_schema, sym):
 
     lmdb_version_store_dynamic_schema.write(sym + "dynamic_schema", date_series)
     assert_series_equal(lmdb_version_store_dynamic_schema.read(sym + "dynamic_schema").data, date_series)
+
+@pytest.mark.skip(reason="Insufficient memory in Github action")
+def test_write_max_size_pickled_data(lmdb_version_store_allows_pickling_max_msg_pack_size):
+    symbol = "test_write_max_size_pickled_data"
+    data = bytearray(b'\x24' * ((1 << 32) - 1))
```

Comment:
The argument is already a huge string and the `bytearray` is just an unnecessary holder.